### PR TITLE
#160617627 Setup test environment

### PR DESCRIPTION
**Pull request by kimbugp**
#### What does this PR do?
-Sets up environment for writing tests using jest and enzyme 

#### How should this be manually tested?
- Run `npm run test`
#### Any background context you want to provide?
- Jest and enzyme are the recommended test suites for React 
#### What are the relevant pivotal tracker stories?
- #160617627

[DELIVERS #160617627]

### CODE REVIEW

File: src/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import {configure} from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+configure({adapter:new Adapter()});
+
```
Comment (LehruAngela) [inline src/setupTests.js:5]: This should preferably be in the root folder

**Review by LehruAngela — CHANGES_REQUESTED** [review]

File: src/setupTests.js (hunk 1 above)
Comment (kimbugp) [reply src/setupTests.js:5]: @LehruAngela, when I put it out of the src directory,  It is not recognised by the Enzyme.
It's automatically detected if it's in the src folder

Comment (LehruAngela) [reply src/setupTests.js:5]: It is because the `__tests__` are in the src folder. Please take them to the root directory as well

Comment (kimbugp) [reply src/setupTests.js:5]: Let me work on it

Comment (kimbugp) [reply src/setupTests.js:5]: @LehruAngela ,I have put them in root but ,the test suite doesnt recognise them.It seems like the create-react-app sets predetermined locations for tests 
`/src/**/__tests__/**/*.{js,jsx}` and `/src/**/?(*.)(spec|test).{js,jsx} `

I tried over-riding this but it returns the error below.
screen shot 2018-10-31 at 12 57 28

Comment (LehruAngela) [reply src/setupTests.js:5]: Oh snap, alright let's leave it like that then.

**Pull request merged**